src/gui/guiFormSpecMenu.cpp

```diff
@@ -1704,19 +1720,34 @@ void GUIFormSpecMenu::parseLabel(parserData* data, const std::string &element)
 
 			if (data->real_coordinates) {
 				// Lines are spaced at the distance of 1/2 imgsize.
-				// This alows lines that line up with the new elements
-				// easily without sacrificing good line distance.  If
+				// This allows lines that line up with the new elements
+				// easily without sacrificing good line distance. If
 				// it was one whole imgsize, it would have too much
 				// spacing.
 				v2s32 pos = getRealCoordinateBasePos(v_pos);
 
-				// Labels are positioned by their center, not their top.
-				pos.Y += (((float) imgsize.Y) / -2) + (((float) imgsize.Y) * i / 2);
+				// Add newline spacing
+				pos.Y += (float)imgsize.Y * stof(style.get(StyleSpec::NEWLINE_SPACING,
+					"0.5")) * i;
+
+				// Move the label around for alignment
+				s32 font_width = m_font->getDimension(wlabel_plain.c_str()).Width;
+				s32 font_height = font_line_height(m_font);
```

Comment:
this should be `m_font->getDimension(wlabel_plain.c_str()).Height;` ?

Reply:
I seem to recall having trouble with this; I think it made the vertical rect too short for the text.  Then again, looking at the centered vertlabel in the screenshot, the rect appears a bit too tall.  I'll experiment around with it again.

I don't see how it breaks compat, though...

Reply:
Oh sorry - what I mean is that it assumes there is only one line in the test

Reply:
I suppose that it may not be worth doing anything too drastic here to support multiline labels well, as an area label/textblock is really needed for multiline labels